plugins/dell-k2/fu-dell-k2-ec-hid.c

```diff
@@ -50,18 +50,28 @@ typedef struct __attribute__((packed)) { /* nocheck:blocked */
 } FuEcHIDCmdBuffer;
 
 gboolean
-fu_dell_k2_ec_hid_write(FuDevice *device, GBytes *buf, GError **error)
+fu_dell_k2_ec_hid_write(FuDevice *device, GBytes *buf, gboolean skip_dev_ack, GError **error)
```

Comment:
do you need to handle the skip_dev_ack() at this level? I think it would be better to handle it in fu_dell_k2_ec_write_firmware_helper()